### DISCUSSION TITLE
2005/07/25/%E4%BB%A5%E5%BE%80%E8%AE%A1%E6%95%B0%E5%99%A8%E4%B8%8B%E7%9A%84%E7%95%99%E8%A8%80

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#42] [Announcements]
# 2005/07/25/%E4%BB%A5%E5%BE%80%E8%AE%A1%E6%95%B0%E5%99%A8%E4%B8%8B%E7%9A%84%E7%95%99%E8%A8%80

https://blog.qizhen.xyz/2005/07/25/%E4%BB%A5%E5%BE%80%E8%AE%A1%E6%95%B0%E5%99%A8%E4%B8%8B%E7%9A%84%E7%95%99%E8%A8%80

**Comment by ruanqizhen** [comment]
Qizhen
2005年07月27日 09:32 编辑
每次都要创建一个新的才能保证它置顶。没什么好办法。

回复
浩
2005年07月26日 22:16 编辑
看到你去踩了下我的Spaces，所以我就不请自来了，一见你的Spaces，很简单感觉，呵呵~~！很特别~！欢迎你的继续关注！！希望能和你成为朋友~！！

回复
夏
2005年07月26日 21:41 编辑
一进来就学了不少新东西.请问楼主:这个置顶的计数器怎样弄呢?谢谢:)

回复
